pkg/client/unversioned/portforward/portforward.go

```diff
@@ -173,8 +175,8 @@ func (pf *PortForwarder) forward() error {
 
 // listenOnPort delegates tcp4 and tcp6 listener creation and waits for connections on both of these addresses.
 // If both listener creation fail, an error is raised.
-func (pf *PortForwarder) listenOnPort(port *ForwardedPort) error {
-	errTcp4 := pf.listenOnPortAndAddress(port, "tcp4", "127.0.0.1")
+func (pf *PortForwarder) listenOnPortBothIpv4AndIpv6(port *ForwardedPort, hostname string) error {
+	errTcp4 := pf.listenOnPortAndAddress(port, "tcp4", hostname)
```

Comment:
ipv4 only?  This behavior is weird and not what I'd expect if I ran this command.